src/main/java/org/sortingmadness/Sorter.java

```diff
@@ -18,34 +25,69 @@ public void setSortingStrategy(SortingStrategyEnum strat) {
         }
     }
 
+    /**
+     * Metoda getAlgorithm() zwraca obiekt algorytmu sortowania
+     * @return SortingAlgorithm
+     */
     public SortingAlgorithm getAlgorithm() {
         return this.algorithm;
     }
 
+    /**
+     * Metoda setSortingAlgorithmDirection() ustawia kierunek sortowania
+     * @param isAscending - kierunek sortowania
+     */
     public void setSortingAlgorithmDirection(Boolean isAscending) {
         this.algorithm.ascending = isAscending;
     }
 
+    /**
+     * Metoda setSortingAlgorithmLimit() ustawia limit sortowania
+     * @param limit - limit sortowania
+     */
     public void setSortingAlgorithmLimit(int limit) {
         this.algorithm.limit = limit;
     }
 }
 
-// interfejs SortingAlgorithm jest wspólny dla wszystkich algorytmów sortowania
+/**
+ * Interfejs SortingAlgorithm jest wspólny dla wszystkich algorytmów sortowania
+ * @param ascending - kierunek sortowania
+ * @param limit - limit sortowania
+ */
 interface SortingAlgorithm {
     public Boolean ascending = true;
```

Comment:
Tu jest duży problem, bo Interfesjy nie mogą mieć zmiennych. Musimy to przebobić żeby był setter. Np. setLimit(int) i setAscending(Boolean)